polling/infrequent/worker/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/temporalio/samples-go/polling"
+	"log"
+
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/worker"
+
+	"github.com/temporalio/samples-go/polling/infrequent"
+)
+
+func main() {
+	// The client and worker are heavyweight objects that should be created once per process.
+	c, err := client.Dial(client.Options{
+		HostPort: client.DefaultHostPort,
+	})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	w := worker.New(c, "pollingSampleQueue", worker.Options{})
```

Comment:
Use a different task queue name for each sample. Otherwise, it might create confusion if a worker for another sample is still running.